code/site/components/com_pages/dispatcher/behavior/decoratable.php

```diff
@@ -30,13 +30,16 @@ protected function _beforeSend(KDispatcherContextInterface $context)
 
     public function getDecorator()
     {
-        $content = $this->getResponse()->getContent();
+        $result = false;
```

Comment:
@johanjanssens Couldn't find where you use `false` as a result of the decoration.

Reply:
@ercanozkaya Here: https://github.com/joomlatools/joomlatools-framework/blob/master/code/libraries/joomlatools/component/koowa/dispatcher/behavior/decoratable.php#L96 I need to ensure that this returns something else then joomla, to prevent double rendering. 

What this code bascially says is: if there is no content in the response i cannot tell you how to decorate, so i’m returning false, aka no decoration at all.